nuxeo-core/nuxeo-core-event/src/main/java/org/nuxeo/ecm/core/event/pipe/queue/QueueBaseEventBundlePipe.java

```diff
@@ -110,6 +110,7 @@ public void run() {
                         }
                     } catch (InterruptedException e) {
                         consumerTPE.shutdown();
+                        Thread.currentThread().interrupt();
```

Comment:
Please move the `interrupt` right after the catch so that it cannot get lost in the event of the other method failing.